### ISSUE TITLE
dataset location option

### ISSUE THREAD

**Issue by ajh3000** [r-dbi/bigrquery#246]
My "bigquery" dataset is located in northeast-asia-1, not the default "unspecified".

I'm new to Big Query.  Through the Google Cloud console, I was able to create tables in Big Query and query them but only after specifying "location".  There is also Google CLI option --location.

https://cloud.google.com/bigquery/docs/dataset-locations

Just wondering how I should specify location through _bigrquery_.  I can't see that option in the doco.

**Comment by MiloParigi** [comment]
You can simply pass properties in the `...` argument of each funtion.

In this case, the documentation you provided said:

> When you use the API, specify your region in the location property in the jobReference section of the job resource.

So just specify a `location` argument with the location you want (so either EU or US), like so:

`bq_dataset_create(bq_dataset_object, location = "EU")`

Hope that help !

**Comment by hadley** [comment]
Thanks @MiloParigi!

**Comment by jeroenclaes** [comment]
The fix suggested in this thread does no longer work. I am using the latest version from CRAN, but I get the following error when I try to create a dataset

`bq_dataset_create(paste0(projectName, ".", dataSetName), location='EU')`

`Error: Invalid JSON payload received. Unknown name "location" at 'dataset': Proto field is not repeating, cannot start list. [badRequest]`

**Comment by hadley** [comment]
@jeroenclaes this is a different problem, so in future please open a new issue. That said, I've already fixed the problem in the dev version,

**Comment by jeroenclaes** [comment]
Oops, sorry for that. Thanks for the fix!
